server/sonar-server/src/test/java/org/sonar/server/user/ws/ChangePasswordActionTest.java

```diff
@@ -57,10 +57,11 @@
 
 public class ChangePasswordActionTest {
 
+  private System2 system2 = mock(System2.class);
```

Comment:
no use is actually made of this mock

especially, the property is hidden by a variable with the same name (but storing an instance of `System2`) in method `setUp`